Makefile

```diff
@@ -202,47 +188,12 @@ install: update-lock ensure-qpdf ensure-system-dependencies
 	@echo "Installing dependencies..."
 	@$(POETRY) install --no-root --sync
 
-# Add dependencies from base-requirements.txt
-.PHONY: add-dependencies
-add-dependencies: ensure-poetry
-	@echo "Adding dependencies from $(REQUIREMENTS_FILE)..."
-	@if [ -f $(REQUIREMENTS_FILE) ]; then \
-		while read -r line; do \
-			if [[ $$line != \#* && -n $$line ]]; then \
-				$(POETRY) add $$line || echo "Failed to add: $$line"; \
-			fi \
-		done < $(REQUIREMENTS_FILE); \
-	else \
-		echo "$(REQUIREMENTS_FILE) not found. Skipping dependency addition."; \
-	fi
-
-# Set up Enterprise Knowledge Retriever project using pip and run the app
-.PHONY: ekr
-ekr: start-parsing-service install-python-versions
-	@echo "Setting up Enterprise Knowledge Retriever project..."
-	@cd $(EKR_DIR) && ( \
-		echo "Current directory: $(shell pwd)"; \
-		echo "EKR_DIR: $(EKR_DIR)"; \
-		echo "EKR_VENV: $(EKR_VENV)"; \
-		if [ ! -d $(EKR_VENV) ]; then \
-			echo "Creating new virtual environment for EKR using Python $(EKR_PYTHON_VERSION)..."; \
-			$(PYTHON) -m venv $(EKR_VENV); \
-		else \
-			echo "Using existing virtual environment for EKR."; \
-		fi; \
-		echo "Activating virtual environment: $(EKR_VENV_ACTIVATE)"; \
-		. $(EKR_VENV_ACTIVATE) && \
-		echo "Upgrading pip..."; \
-		$(PIP) install --upgrade pip && \
-		if [ -f requirements.txt ]; then \
-			echo "Installing requirements from requirements.txt..."; \
-			$(PIP) install -r requirements.txt; \
-		else \
-			echo "requirements.txt not found in $(EKR_DIR). Skipping dependency installation."; \
-		fi && \
-		echo "Starting EKR application..."; \
-		$(EKR_COMMAND); \
-	)
+# Post-process installation
+.PHONY: post-process
```

Comment:
This is because there seems to be a known bug with google-search-results and make, for some reason it needs a reinstall due to an init being overwritten so this is what this small fix is doing.